display/download.go

```diff
@@ -321,8 +321,15 @@ func downloadPage(p *structs.Page) (string, error) {
 func downloadNameFromURL(dir, u, ext string) (string, error) {
 	var name string
 	var err error
+
 	parsed, _ := url.Parse(u)
-	if parsed.Path == "" || path.Base(parsed.Path) == "/" {
+	if strings.HasPrefix(parsed.String(), "about:") {
+		// Is an about page, use the entire page url since there is no hostname
+		name, err = getSafeDownloadName(dir, parsed.String()+ext, true, 0)
```

Comment:
I would just use the "path" or whatever. Like `about:subscriptions` should be saved as `subscriptions.gmi`.